contrib-02-lhcb-openstack/Readme.md

```diff
@@ -0,0 +1,192 @@
+# LHCbSoftware on Openstack
+
+## Motivation
+
+The easiest way to to work in an LHCb Software Evironment is by logging in to an `lxplus` instance.
+From there one also has access to the [`AFS` file storage](http://information-technology.web.cern.ch/services/afs-service).
+
+However, working on `lxplus` comes with a few significant limitations:
+ * No super user access. This makes it hard to install new software.
+ * Shared resources with other users. Others might start resource-hungry processes on the same machine, therefore extending execution time and making it unpredicatable.
```

Comment:
Also, going a bit to the original motivation: limited time/memory for running processes? this means that long profiling sessions may get killed, for example.